sawmill-core/src/test/java/io/logz/sawmill/processors/DeDotProcessorTest.java

```diff
@@ -0,0 +1,124 @@
+package io.logz.sawmill.processors;
+
+import io.logz.sawmill.Doc;
+import io.logz.sawmill.utilities.JsonUtils;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.logz.sawmill.utils.FactoryUtils.createConfig;
+import static io.logz.sawmill.utils.FactoryUtils.createProcessor;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DeDotProcessorTest {
+
+    private final String messageExample = "{\n" +
```

Comment:
let's be more accurate with the JSON
most of the fields here are not relevant for the test